src/constants/common.ts

```diff
@@ -11,7 +11,10 @@ export const PAGE_CELL_COUNT = 200
 export const NEXT_HARD_FORK_EPOCH = 5414
 export const EPOCH_HOURS = 4
 export const ONE_DAY_SECOND = 24 * 60 * 60
+export const ONE_DAY_TIMESTAMP = ONE_DAY_SECOND * 1000
```

Comment:
I think it is clearer to use `ONE_DAY_MILLISECOND` instead of `ONE_DAY_TIMESTAMP`

Reply:
done